### PR TITLE
Update to workspaceTags

### PR DESCRIPTION
**Pull request by cobey**
To better understand the level of adoption of versions of Azure SDKs, we are adding the listed modules to the workspace tagging process.

### CODE REVIEW

File: src/vs/workbench/contrib/tags/electron-browser/workspaceTagsService.ts [hunk 1]
```diff
@@ -99,16 +99,70 @@ const PyMetaModulesToLookFor = [
 
 const PyModulesToLookFor = [
 	'azure',
+	'azure-appconfiguration',
```
Comment (RMacfarlane) [inline src/vs/workbench/contrib/tags/electron-browser/workspaceTagsService.ts:102]: These GDPR comment below needs to be updated with these tags

**Review by RMacfarlane — CHANGES_REQUESTED** [review]
Thanks, sorry for the delay in reviewing! The tags need to be updated as well

**Comment by cobey** [comment]
I have updated to include the GDPR packages

**Review by RMacfarlane — APPROVED** [review]
LGTM!

**Comment by RMacfarlane** [comment]
There are some conflicts with `main` and I don't have permission to edit the PR - @cobey Can you please merge in the latest `main` so that I can merge this in?

**Comment by cobey** [comment]
Superseded by #122468

**Pull request closed without merging**